CONTRIBUTING.md

```diff
@@ -266,6 +266,17 @@ the production server.
 To see deploy logs and more, visit project's [dashboard][] -- Netlify login
 required.
 
+## Approver and Maintainer practices
+
+This last section includes guidelines and some common practices used by approvers and maintainers while doing code reviews:
+
+* PRs with changes to documentation co-owned by a SIG (collector, demo, language-specific...) should aim for two approvals: one by a docs approver and one by a SIG approver:
+  * Doc approver label such PRs with `sig:<name>` and tag the SIG `-approvers` group on that PR
+  * If no SIG approval is given within a certain grace period, docs maintainer may use their own judgement to merge that PR
+* If the PR branch is `out-of-date with the base branch`, they do not need to be updated continously: every update triggers all the PR CI checks to be run! It's often enough to update them before merging.
+* A PR by non-maintainers should **never** update git sub modules. This happens by accident from time to time. Let the PR author know that they should not worry about it, we will fix this before merging, but in the future they should make sure that they work from an up-to-date fork.
```

Comment:
Any CLA related tip or troubleshooting? What if contributors can't figure it out or are stuck?

Reply:
I think for this they need to enable maintainer permissions to update their branch?

Reply:
Yeah, CLA is a thing on it's own. If they did a mistake with their email address in one commit to their PR (most common issue), we can either ask them to fix this themselves or rebase the PR (which requires write permissions as @cartermp said). Sometimes CLA is just behaving strange/unexpected, in the worst case the PR needs to be closed & re-opened to re-trigger the checks